### PR TITLE
Document custom formatting for sentinel alerts (#82)

### PR DESCRIPTION
**Pull request by mok0230**
See also: https://github.com/OpenZeppelin/defender/pull/2282

### CODE REVIEW

**Review by spalladino — APPROVED** [review]

File: docs/modules/ROOT/pages/sentinel.adoc [hunk 1]
```diff
@@ -653,6 +653,102 @@ exports.handler = async function(params) {
...
+
+You can optionally modify the notification message content and formatting using the checkbox below the notification channel selector.
+
+=== Message Syntax
+
+Custom notifications support a limited set of markdown syntax:
+
+* Bold
+* Italic
+* Links
```
Comment (mverzilli) [inline docs/modules/ROOT/pages/sentinel.adoc:667]: Let's add in place examples. Nothing too fancy, something like:

* Bold (`**this text is bold**`)
* Italic (`*this text is italicized*`)

etc

File: docs/modules/ROOT/pages/sentinel.adoc [hunk 2]
```diff
@@ -653,6 +653,102 @@ exports.handler = async function(params) {
...
+
+There is partial support for additional markdown syntax, but rendering behavior varies by platform.
```
Comment (mverzilli) [inline docs/modules/ROOT/pages/sentinel.adoc:669]: If we're running out of time we can leave like this, but ideally I'd like to see a bit more discussion around this. Reading it feels like there *could* be support for some things I might want to use but there's no way to confirm it.

File: docs/modules/ROOT/pages/sentinel.adoc [hunk 3]
```diff
@@ -653,6 +653,102 @@ exports.handler = async function(params) {
...
+In addition to the standard event schema, the following parameters are injected for usage in custom notification messages: 
+
+* `transaction.link`
+* `matchReasonsFormatted`
+
+=== Character Limit
+
+Messages will be truncated if they exceed a platform's character limit. The best practice is to limit messages to 1900 characters.
+
+=== Example
```
Comment (mverzilli) [inline docs/modules/ROOT/pages/sentinel.adoc:684]: Open for discussion, but I'd make the Example section the first one after the introductory sentence ("You can optionally modify the notification message content and formatting using the checkbox below the notification channel selector(...)").

The detailed explanations of syntax and dynamic content will make much more sense after skimming the example

**Review by mverzilli — APPROVED** [review]

File: docs/modules/ROOT/pages/sentinel.adoc (hunk 3 above)
Comment (mok0230) [reply docs/modules/ROOT/pages/sentinel.adoc:661]: makes sense, done

**Review by mok0230 — COMMENTED** [review]
@mverzilli pushed code and responded to your comments - see above

File: docs/modules/ROOT/pages/sentinel.adoc (hunk 2 above)
Comment (mok0230) [reply docs/modules/ROOT/pages/sentinel.adoc]: I thought about just including the cue that they will see a warning if using any features with partial support, but provided a little more context than that. If it feels too wordy now, I can cut it down - let me know what you think.
